test-network-function/networking/suite.go

```diff
@@ -202,7 +202,7 @@ func runNetworkingTests(netsUnderTest map[string]netTestContext, count int) map[
 func testDefaultNetworkConnectivity(env *config.TestEnvironment, count int) {
 	ginkgo.When("Testing Default network connectivity", func() {
 		testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestICMPv4ConnectivityIdentifier)
-		ginkgo.It(testID, func() {
+		ginkgo.It(testID, ginkgo.Label(testID), func() {
```

Comment:
This test should not have the same label as the multus test. The test is also defined with the context hierarchy e.g. with ginkgo.When.
there is a function to do this automatically (used for claim.json) but it would make the label too long -> ContainerHierarchyTexts

See:
networking Both Pods are on the Default network Testing Default network connectivity
  networking-icmpv4-connectivity [**networking-icmpv4-connectivity**]
  /home/deliedit/redhat/github/david/upstream/test-network-function/networking/suite.go:205
networking Both Pods are connected via a Multus Overlay Network Testing Multus network connectivity
  networking-icmpv4-connectivity [**networking-icmpv4-connectivity**]

Reply:
Actually, shouldn't they use a different test id? If that is the only case, it might be easier to change the multus test id to be something like networking-icmpv4-connectivity-multus